tests/migrations/test_commands.py

```diff
@@ -462,6 +463,21 @@ def test_migrate_record_squashed(self):
         )
         # No changes were actually applied so there is nothing to rollback
 
+    @override_settings(MIGRATION_MODULES={'migrations': 'migrations.test_migrations_inconsistent_history'})
```

Comment:
Do you need that test migration app or can you use another already existing one?